src/GeneticSharp.Domain/OperatorsStrategies/DefaultOperatorsStrategy.cs

```diff
@@ -0,0 +1,55 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using GeneticSharp.Domain.Chromosomes;
+using GeneticSharp.Domain.Crossovers;
+using GeneticSharp.Domain.Mutations;
+using GeneticSharp.Domain.Randomizations;
+
+namespace GeneticSharp.Domain
+{
+    public class DefaultOperatorsStrategy : IOperatorsStrategy
+    {
+        /// <summary>
+        /// Crosses the specified parents.
+        /// </summary>
+        /// <param name="crossover">The crossover class.</param>
+        /// <param name="crossoverProbability">The crossover probability.</param>
+        /// <param name="parents">The parents.</param>
+        /// <returns>The result chromosomes.</returns>
+        public IList<IChromosome> Cross(ICrossover crossover, float crossoverProbability, IList<IChromosome> parents)
+        {
+            var offspring = new List<IChromosome>();
+
+            for (int i = 0; i < parents.Count; i += crossover.ParentsNumber)
```

Comment:
Changed loop use parents.Count, previously was using population. Please confirm it's the same.

Reply:
It's not the same, because in this code on GeneticAlgorithm class: 
```csharp
private bool EvolveOneGeneration()
{
   var parents = SelectParents();
   var offspring = Cross(parents);
```
The parents are selected by the Selection operator, so, the parents list passes to the Crossover operator is probably smaller than Population.MinSize.

Reply:
It's still using parents.Count, instead of population.MinSize.

Reply:
Not changed, it's still using parents.Count, instead of population.MinSize.